src/main/java/frc/robot/commands/CurvatureDriveWithJoystick.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import frc.robot.config.Config;
+/**
+ * Allows the robot to be driven using curve drive
+ */
+public class CurvatureDriveWithJoystick extends CurvatureDrive {
+
+    /**
+     * Drives using axes that are on separate joysticks
+     *
+     * @param joy1    The joystick that the first axis is on
+     * @param axis1   The first axis
+     * @param invert1 Whether to negate the first joystick input
+     * @param joy2    The joystick that the second axis is on
+     * @param axis2   The second axis
+     * @param invert2 Whether to negate the second joystick input
+     */
+    public CurvatureDriveWithJoystick(Joystick joy1, int axis1, boolean invert1,
```

Comment:
If the values for invert are always the same you should consider not passing them and simply inferring them here.